src/vkutil.h

```diff
@@ -13,9 +13,9 @@
  *   * Redistributions in binary form must reproduce the above copyright
  *     notice, this list of conditions and the following disclaimer in the
  *     documentation and/or other materials provided with the distribution.
- *   * Neither the name of Redis nor the names of its contributors may be used
```

Comment:
The commit message describes https://github.com/valkey-io/libvalkey/issues/48 only, is more description needed?